boards: arm: thingy53: Disable USB remote wakeup by default + fix setting default USB related log lvl

mkapala-nordic opened the pull request:
For Thingy:53 board, USB remote wakeup should be disabled by default. 
Current solution disabled it from Kconfig.defconfig, but was always overwritten by imply in CONFIG_USB_NRFX. 
This change disables it in board *_defconfig files.
Fixes: https://github.com/zephyrproject-rtos/zephyr/issues/54702

Previous method of setting default USB related log levels for Thingy53
led to generating inconsistent configuration, ex.
CONFIG_USB_CDC_ACM_LOG_LEVEL=0 and CONFIG_USB_CDC_ACM_LOG_LEVEL_INF=y
at the same time. This commit changes to proper implementation
which is overlaying choice's default.
Fixes: https://github.com/zephyrproject-rtos/zephyr/issues/54703

Signed-off-by: Mateusz Kapala <mateusz.kapala@nordicsemi.no>

mkapala-nordic committed:
boards: arm: thingy53: Disable USB remote wakeup by default

For Thingy:53 board, USB remote wakeup should be disabled by
default. Current solution disabled it from Kconfig.defconfig,
but was always overwritten by imply in CONFIG_USB_NRFX.
This change disables it in board *_defconfig files.

Signed-off-by: Mateusz Kapala <mateusz.kapala@nordicsemi.no>

MarekPieta commented on boards/arm/thingy53_nrf5340/thingy53_nrf5340_cpuapp_defconfig:
```diff
@@ -24,3 +24,5 @@ CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 
 CONFIG_PINCTRL=y
+
```
Consider adding comments in these configuration files explaining that we enable USB by default in `Kconfig.defaults`, but because of imply in the USB nrfx driver the remote wakeup must be disabled here (looking only at this file the change may look a bit out of context).

mkapala-nordic committed:
boards: arm: thingy53: Fix default USB related log levels

Previous method of setting default USB related log levels for Thingy53
led to generating inconsistent configuration, ex.
CONFIG_USB_CDC_ACM_LOG_LEVEL=0 and CONFIG_USB_CDC_ACM_LOG_LEVEL_INF=y
at the same time. This commit changes to proper implementation
which is overlaying choice's default.

Signed-off-by: Mateusz Kapala <mateusz.kapala@nordicsemi.no>

nordicjm commented:
@mkapala-nordic can you create a bug for this and link it with `Fixes #<ID>` in the first post? That way the target can be set to 3.3

nordicjm commented:
@stephanosio this can be merged

stephanosio commented:
@anangl Can you please take a quick look since you are the assignee?
